.github/workflows/ci.yaml

```diff
@@ -0,0 +1,17 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  tox:
+    runs-on: ubuntu-20.04
```

Comment:
Should this be `ubuntu-latest` instead?

according to https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions, it also points to focal, but then we will not need to change this string every time a new LTS is out.

Reply:
Hi Athos! Normally it's good practice to "pin" the dependencies and in general the external environment where the tests are made, so the CI or test run won't start failing because something else changed. Then dependencies are updated manually when needed, and checking that nothing breaks.

Here we're not really doing it, as `tox.ini` just pulls the latest versions of the linters, and indeed the tox test run broke in `master`. I think this is fine in this specific case, as the project is small and mostly internal, and if CI breaks we can just fix it without really blocking anybody's work. Which means: we could also use `ubuntu-latest` for the same reason. But as this PR landed already let's keep Focal for now, IMHO.